recipes/pypy/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pypy2" %}
+{% set version = "5.10.0" %}
+{% set hash = "c966124497ba8728654ce1161fa4c46b035ff30f289be70960f58292e5897cc8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/squeaky/portable-pypy/downloads/pypy-5.10.0-linux_x86_64-portable.tar.bz2
+  fn: {{ name|lower }}_{{ version }}.tar.bz2
+  sha256: {{ hash }}
+
+
+build:
+    number: '0'
+    skip: True  # [not osx]
+
+requirements:
+    run:
+        - openssl
```

Comment:
Some of these will probably need to be have version constraints, depending on how the upstream package is built. I'll investigate later.